core/config_help.json

```
@@ -300,6 +300,27 @@
       "See also: `thread_close_title`, `thread_close_footer`, `thread_close_response`."
     ]
   },
+  "thread_move_notify": {
+    "default": "No",
+    "description": "Notify the recipient if the thread was moved.",
+    "examples": [
+      "`{prefix}config set thread_move_notify yes`",
+      "`{prefix}config set thread_move_notify no`"
+    ],
+    "notes": [
+      "See also: `thread_move_response`"
+    ]
+  },
+  "thread_move_response": {
+    "default": "This thread has been moved.",
+    "description": "This is the message to display to the user when the thread is moved.",
+    "examples": [
+      "`{prefix}config set thread_move_response This thread has been moved to another category for review!`"
+    ],
+    "notes": [
+      "Only has an effect when thread_move_notify is on."
```

```suggestion
      "Only has an effect when `thread_move_notify` is on.",
      "See also: `thread_move_notify`."
```

Add a "see also" back reference.